CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 - Added support for resolving superclass properties for not-NSObject subclasses
 - The `{% for %}` tag can now iterate over tuples, structures and classes via
   their stored properties.
+- Added `break` and `continue` tags to break or continue current loop
```

Comment:
Please conform these to the formatting for other changelog entries (`.`, 2 spaces, credit yourself and link to this PR).